libs/community/tests/integration_tests/test_docai.py

```diff
@@ -0,0 +1,19 @@
+"""Integration tests for the Google Cloud DocAI parser."""
+
+import os
+
+import pytest
+from langchain_core.document_loaders.blob_loaders import Blob
+
+from langchain_google_community.docai import DocAIParser
+
+
```

Comment:
as the test requires external gcp components could you please annotate it with @pytest.mark.extended

Reply:
Annotated